src/frontend/js/media-management/component/edit-sidebar.tsx

```diff
@@ -253,6 +254,40 @@ export default function EditSidebar({
                       {mediaTranslations.btn_save_file}
                     </button>
                   )}
+     
```

Comment:
Here, you would have to add a new `<form>`-tag for the new `ReplaceMediaFileForm`.